src/main/java/nl/tudelft/pixelperfect/Constants.java

```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.pixelperfect;
+
+/**
+ * A data class for storing game-wide items that can all be altered in a single sweep.
+ * 
+ * @author David Alderliesten
+ *
+ */
+public final class Constants {
+  // Gui-styling related constants for the in-game HUD.
+  public static final int guiWidthOffset = 150;
+  public static final int guiHeightOffset = 50;
```

Comment:
Indeed, but an exception is allowed to be made for a constants class, see the lack of checkstyle warnings about it.